appveyor.yml

```diff
@@ -1,8 +1,8 @@
 os: Visual Studio 2015
 
 environment:
-  global:
-    node_pre_gyp_mock_s3: true
+  #global:
+  #  node_pre_gyp_mock_s3: true
```

Comment:
Can this commented code be removed now since we declare the right env per job and the defaults are reasonable?

Reply:
i missed these when uncommenting the travis.yml definitions. i think i need to uncomment them so they won't fail on an external PR. i can test by creating an external PR that just uncomments them; if they don't fail then i'll remove them.

Reply:
i guess i was missing something. removed.